api/src/main/java/org/pmiops/workbench/db/dao/DataSetServiceImpl.java

```diff
@@ -591,6 +590,16 @@ private ValuesLinkingPair getValueSelectsAndJoins(List<DomainValuePair> domainVa
     return new ValuesLinkingPair(valueSelects, valueJoins);
   }
 
+  private static String toTitleCase(String name) {
+    if (name.isEmpty()) {
+      return name;
+    } else if (name.length() == 1) {
+      return name;
+    } else {
+      return String.format("%s%s", name.charAt(0), name.substring(1).toLowerCase());
```

Comment:
Nit: I would explicitly set the first character to uppercase, just for documentation reasons/input safety

Reply:
good point.